src/core/server/saved_objects/migrations/kibana/kibana_migrator.ts

```diff
@@ -107,24 +107,15 @@ export class KibanaMigrator {
    *    The promise resolves with an array of migration statuses, one for each
    *    elasticsearch index which was migrated.
    */
-  public runMigrations(skipMigrations: boolean = false): Promise<Array<{ status: string }>> {
+  public runMigrations(): Promise<Array<{ status: string }>> {
```

Comment:
👍 this `execute(dontExecute: boolean)` wasn't making any sense.

Reply:
Haha, I totally got a "WTF?" moment even though I wrote this code myself.